doc/languages-frameworks/php.section.md

```diff
@@ -0,0 +1,72 @@
+# PHP
+
+## User Guide
+
+### Using PHP
+
+#### Overview
+
+Several versions of PHP is available on Nix, as well as a bunch of extensions
+and libraries available for each version of PHP.
+
+The attribute `php` refers to the latest version of PHP that has undergone
+sufficient testing in nixpkgs. This may not be the latest and we tend to upgrade
+to the latest after branch-off for a new NixOS release so we get more testing
+before the next release.
+
+We keep releases as long as they are supported for the entirety of the next
+NixOS release life cycle. See [PHP Supported Versions](https://www.php.net/supported-versions.php).
+
+As for packages we have `php.phpPackages` that contains a bunch of attributes where
```

Comment:
`php.phpPackages` feels too verbose. Could we just do `php.packages` or `php.pkgs` like Python does?

Reply:
Good idea! I'll fix that right away.

Reply:
Done!